src/Symfony/Installer/NewCommand.php

```diff
@@ -111,8 +111,8 @@ protected function checkSymfonyVersionIsInstallable()
         }
 
         // validate semver syntax
-        if (!preg_match('/^2\.\d(?:\.\d{1,2})?(?:-(?:dev|BETA\d*|RC\d*))?$/i', $this->version)) {
-            throw new \RuntimeException('The Symfony version must be 2.N or 2.N.M (where N and M are positive integers). The special "-dev", "-BETA" and "-RC" versions are also supported.');
+        if (!preg_match('/^[23]\.\d(?:\.\d{1,2})?(?:-(?:dev|BETA\d*|RC\d*))?$/i', $this->version)) {
+            throw new \RuntimeException('The Symfony version must be 2.N or 2.N.M or 3.N or 3.N.M (where N and M are positive integers). The special "-dev", "-BETA" and "-RC" versions are also supported.');
```

Comment:
I'm not sure but maybe rewording the error message as follows would make it more readable:

```
[...] must be 2.N, 2.N.M, 3.N or 3.N.M [...]
```